website/source/docs/providers/kubernetes/r/secret.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_secret"
+sidebar_current: "docs-kubernetes-resource-secret"
+description: |-
+  The resource provides mechanisms to inject containers with sensitive information while keeping containers agnostic of Kubernetes.
+---
+
+# kubernetes_secret
+
+The resource provides mechanisms to inject containers with sensitive information, such as passwords, while keeping containers agnostic of Kubernetes.
+Secrets can be used to store sensitive information either as individual properties or coarse-grained entries like entire files or JSON blobs.
+
```

Comment:
As this is sensitive topic, I feel we should take an extra step to educate the user and perhaps link to security properties and risks related to this resource?
https://kubernetes.io/docs/user-guide/secrets/#security-properties